chrome-logic-gmail.js

```diff
@@ -186,7 +186,7 @@ function fetchMessagesList(onSuccess, args) {
             'contentType': 'json'
         };
         fetch(
-            `https://www.googleapis.com/gmail/v1/users/${encodeURIComponent(userId)}/messages?key=AIzaSyC02-izGdNpzEBfI4OHfAc6sTIuIst4wXo${qp}`, init)
+            `https://www.googleapis.com/gmail/v1/users/${encodeURIComponent(userId)}/messages?${qp}key=<ENTER_YOUR_API_KEY>`, init)
```

Comment:
This change does not change anything. The ampersand `&` is not required before the first query parameter.

Reply:
Read carefully: in your last commit ampersand & is missing before the first ${qp}. It may lead to a problem if Google change endpoint logic

Reply:
You are right, there is a `&` missing, but it's not where you add it in your changes.
I fixed it at [this commit](https://github.com/lambrospetrou/gmail-keep-importer/commit/dc28e844657aaa83c2772a3cb901680fed295719).

Thanks

Reply:
Actually, your {qp} string generate sequence with ampersand on the end.

Reply:
I specially move "key" parameter to the right side because users may remove ampersand :)
But it's not a solution too